Code/JavaWrappers/ChemTransforms.i

```diff
@@ -67,14 +80,25 @@ RDKit::ROMol *fragmentMolOnBonds(
 
 %newobject fragmentMolOnBonds;
 %ignore fragmentOnBonds;
+%ignore molzip;
 %rename("fragmentOnBonds") fragmentMolOnBonds;
+%rename("molzip") new_molzip;
 
 RDKit::ROMol *fragmentMolOnBonds(
     const RDKit::ROMol &mol, const std::vector<int> &bondIndices,
     bool addDummies = true, 
     const std::vector<std::pair<unsigned int, unsigned int>> *dummyLabels = nullptr,
     std::vector<int> *nCutsPerAtom = nullptr);
 
+RDKit::ROMol * new_molzip(
```

Comment:
Don't you need to also add `%newobject`s for these functions?

Reply:
Added